axelor-account/src/main/java/com/axelor/apps/account/service/payment/invoice/payment/InvoicePaymentValidateServiceImpl.java

```diff
@@ -76,6 +82,7 @@ public InvoicePaymentValidateServiceImpl(
     this.invoicePaymentRepository = invoicePaymentRepository;
     this.reconcileService = reconcileService;
     this.invoicePaymentToolService = invoicePaymentToolService;
+    this.accountConfigRepo = accountConfigRepo;
```

Comment:
remove this injection, instead of using accountConfigRepo directly, use instead accountConfigService.getAccountConfig(company)